lib/presenters/artefact_presenter.rb

```diff
@@ -141,26 +138,6 @@ def nodes
     {"nodes" => presented_nodes}
   end
 
-  def licence
-    return {} unless @artefact.licence
-
-    {
-      "licence" => ArtefactLicencePresenter.new(@artefact.licence).present
-    }
-  end
-
-  def local_service
```

Comment:
Licences and LocalTransactions used to inject the local service information into the response hash. Both have been changed to not use the local service info anymore (LocalTransaction has been migrated and Licences now gets the relevant information straight from Licensify) so it can be removed from the presenter. I added some text to the commit message, does that make it clear? Or let me know if you prefer another commit for this...